driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -1295,6 +1296,24 @@ public Builder withNettyOptions(NettyOptions nettyOptions) {
             return this;
         }
 
+        /**
+         * Sets whether or not the <code>NO_COMPACT</code> startup option should be present and set to true.
+         * <p>
+         * When this option is supplied, <code>SELECT</code>, <code>UPDATE</code>, <code>DELETE</code> and
+         * <code>BATCH</code> statements on <code>COMPACT STORAGE</code> tables function in "compatibility" mode which
+         * allows seeing these tables as if they were "regular" CQL tables.
+         * <p>
+         * This option only effects interactions with tables using <code>COMPACT STORAGE<code>.
+         *
+         * @param noCompact Whether or not NO_COMPACT should be configured on connections.
+         * @return this builder.
+         * @see <a href="https://issues.apache.org/jira/browse/CASSANDRA-10857">CASSANDRA-10857</a>
+         */
+        public Builder withNoCompact(boolean noCompact) {
```

Comment:
Wasn't sure if I should have done it this way or to simply have `withNoCompact()` which enables the flag.  The advantage of doing it this way if you can disable the option after the fact, but the downside is you have to provide the boolean value.

Reply:
I'd prefer the no-argument version, in order to follow the same pattern as `withoutMetrics`.